ethexe/contracts/test/Base.t.sol

```diff
@@ -258,19 +260,16 @@ contract Base is POCBaseTest {
         );
     }
 
+    // TODO: add SignatureType as param here
```

Comment:
should be added now for tests?

Reply:
We may be able to remove the ECDSA in the future. Right now we're testing this in Rust code anyway.